externs/shaka/player.js

```diff
@@ -528,9 +528,7 @@ shaka.extern.AdvancedDrmConfiguration;
  *   <i>Optional.</i> <br>
  *   A dictionary which maps key system IDs to advanced DRM configuration for
  *   those key systems.
- * @property
- *     {((function(!Uint8Array, ?shaka.extern.DrmInfo):!Uint8Array)|undefined)}
- *   initDataTransform
+ * @property {((function(!Uint8Array, ?shaka.extern.DrmInfo):!Uint8Array)|undefined)} initDataTransform
```

Comment:
Is this the reason you changed the max line length?  Please revert this change.

Reply:
Indeed it is, Doctrine does not seem to handline line breaks in type annotations well, I will dig further to fix the actual issue and get rid of this nasty workaround

Reply:
Also, unfortunately, I started working on this pull request long before eslint announced they will deprecate the doctrine parser. As a workaround, I switched to the fork by @teppeis, but there are alternative jsdoc parsers with active development. https://github.com/jsdoctypeparser/jsdoctypeparser is used by eslint-plugin-jsdoc, the recommended successor to eslint's own jsdoc support.
I will investigate whether it'd make sense to exchange doctrine with jsdoctypeparser or if it's sufficient to fix this issue in doctrine itself.

Reply:
jsdoctypeparser also does not handle linebreaks in type annotations, so I will investigate the issue in doctrine instead

Reply:
I think I have nailed down the actual issue. It is not the newlines themselves that cause the problem, but where they occur. Doctrine expects the beginning of the type directly without a newline, but once the type has started, newlines can occur inside. I have adapted the code accordingly. Furthermore, I have found actual syntax errors (which surprisingly weren't picked up by the closure compiler) in other type declarations by configuring the typescript generator to actually throw once syntax errors are encountered in the comments.